src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -121,12 +129,52 @@ def init_from_file(self) -> None:
 
         self.profiles = profile_jsonc.get("profiles", {})
         self.defaults = profile_jsonc.get("defaults", {})
+        self.schema_property = profile_jsonc.get("$schema", None)
 
         # loading secure props is done in load_profile_properties
         # since we want to try loading secure properties only when
         # we know that the profile has saved properties
         # self.load_secure_props()
 
+    def schema_list(
+        self,
+    ) -> list:
+        """
+        Loads the schema properties
+        in a sorted order according to the priority
+        
+        Returns
+        -------
+        Dictionary
+        
+            Returns the profile properties from schema (prop: value)
+        """
+
+        schema = self.schema_property
+        if schema is None:
+            return []
+
+        if schema.startswith("https://") or schema.startswith("http://"):
+            schema_json = requests.get(schema).json()
+
+        elif os.path.isfile(schema):
+            with open(schema) as f:
```

Comment:
The schema won't always be in the current working directory. We should evaluate the schema path relative to the location of the `zowe.config.json` file which defines "$schema".
Something like this should work: `os.path.relpath(schema, self.location)`

Reply:
Yes, you are right it can be the `cwd` :blush: